### PR TITLE
[FIX] Typos and Translation Mistakes

### PR DESCRIPTION
**Pull request by hfeky**
@RocketChat/android

#### Changes:

This pull request brings some fixes to typos, sentence structure and translation mistakes.

1. Changed Github to GitHub
2. Changed Gitlab to GitLab
3. Changed Linkedin to LinkedIn
4. Changed Wordpress to WordPress
5. Replaced the string resource name `msg_forgot__your_password` with `msg_forgot_your_password`
6. Replaced the string resource name `msg__your_2fa_code` with `msg_your_2fa_code`
7. Replaced the string resource name `msg_profile_update_successfully` with `msg_profile_updated_successfully`
8. Added a missing question mark in `msg_delete_description` string resource in some translations
9. Fixed some Arabic translation mistakes
10. Fixed question mark direction in some strings in Arabic and Persian strings resource files
11. Translated 4 strings in Spanish strings resource file
12. Changed statics to statistics and improving to improve in preferences messages section in the default strings resource file
13. Fixed the string resource `max_file_size_exceeded` not taking the expected two arguments in Arabic strings resource file

### CODE REVIEW

**Comment by philipbrito** [comment]
@HusseinElFeky Could you fix the conflicts here?

**Comment by hfeky** [comment]
@filipedelimabrito I have merged the branches. Please check if any further change is required.

File: app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt [hunk 1]
```diff
@@ -6,11 +6,7 @@ import android.content.Intent
 import android.graphics.Bitmap
 import android.os.Build
 import android.os.Bundle
-import android.view.LayoutInflater
-import android.view.Menu
-import android.view.MenuItem
-import android.view.View
-import android.view.ViewGroup
+import android.view.*
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt]: @HusseinElFeky Could you remove this wildcard import, please?

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Comment by hfeky** [comment]
@filipedelimabrito I removed the wildcard import. Thanks for pointing it out.

**Review by philipbrito — APPROVED** [review]
Thank you very much for making it @HusseinElFeky 💯

**Pull request merged**